recipe/build.sh

```diff
@@ -61,7 +61,7 @@ cmake ${CMAKE_ARGS} \
     -DBUILD_TESTING=ON                \
     -DCMAKE_INSTALL_LIBDIR=lib        \
     -DCMAKE_INSTALL_PREFIX=${PREFIX}  \
-    -DPython_INCLUDE_DIR=$(python -c "from sysconfig import get_paths as gp; print(gp()['include'])") \
+    -DPython_INCLUDE_DIR=$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])") \
```

Comment:
Might need
```suggestion
    -DPython_INCLUDE_DIR="%PREFIX/include"
```
if I cannot execute the cross-architecture `$PYTHON` here in the configure/build step.